docs/guide/dev/advanced_installation/README.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
This file lost history. Original path `guides/dev/advanced_installation/README.md`.